src/coreclr/jit/emitarm64.h

```diff
@@ -482,6 +482,40 @@ static code_t insEncodeSveElemsize(emitAttr size);
 // This specifically encodes the field 'tszh:tszl' at bit locations '22:20-19'.
 static code_t insEncodeSveElemsize_tszh_22_tszl_20_to_19(emitAttr size);
 
+// Returns the first register list size for the given SVE instruction.
+static int insGetSveReg1ListSize(instruction ins);
+
+// Returns the predicate type for the given SVE instruction's second register.
+static PredicateType insGetSveReg2PredicateType(insFormat fmt);
+
+// Returns the predicate type for the given SVE instruction's second register.
+static PredicateType insGetSveReg2PredicateType(instruction ins, insFormat fmt);
```

Comment:
since this is not being used, lets remove it and add when needed.

Reply:
makes sense